.github/scripts/push-firefox-bundle-script.sh

```diff
@@ -9,7 +9,16 @@ fi
 
 git config --global user.name "MetaMask Bot"
 git config --global user.email metamaskbot@users.noreply.github.com
-version=$(git show -s --format='%s' HEAD | grep -Eo 'v[0-9]+\.[0-9]+\.[0-9]+')
+rawVersion=$(git show -s --format='%s' HEAD | grep -Eo 'release/[0-9]+\.[0-9]+\.[0-9]+' | sed 's|release/||')
+version="v${rawVersion}"
+
+# Validate that the version was successfully extracted
+if [[ -z "${rawVersion}" ]]; then
+    echo "::error:: Failed to extract version from commit message. Ensure it follows the 'release/x.y.z' format."
+    exit 1
+fi
+
+echo "Version extracted: ${version}"
```

Comment:
All changes in this file come from this PR https://github.com/MetaMask/metamask-extension/pull/36291, which was not included in 13.4.0, but was cherry-picked in 13.3.2.